recipe/meta.yaml

```diff
@@ -1,50 +1,44 @@
 {% set name = "hkl" %}
-{% set version = "5.0.0.2173" %}
-{% set sha256 = "5f3991d2af4ff01d6f6e29d0176af9bc1b54e6a8598046a050ad804fedd860f2" %}
+{% set gitorg = "bluesky" %}
+{% set version = "v5.0.0.3001" %}
+{% set license = "COPYING" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/picca/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: {{ sha256 }}
-  patches:
-    - no-docs.patch
+  git_rev: main
+  git_url: https://github.com/{{ gitorg }}/libhkl.git
```

Comment:
We should use GH releases' artifacts for that in the future. I opened an issue to document it: https://github.com/bluesky/libhkl/issues/3.

Reply:
Agreed, [as noted](https://github.com/bluesky/libhkl/issues/3#issuecomment-1741089648).